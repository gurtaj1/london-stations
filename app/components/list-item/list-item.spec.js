import React from 'react';
import { shallow } from 'enzyme';

import ListItem from 'components/list-item/list-item';
import fixture from 'components/list-item/list-item.fixture';

describe('<ListItem />', () => {
  const props = fixture;

  it('should export a class', () => {
    expect(ListItem).toBeDefined();
  });

  it('should render as expected, without crashing', () => {
    const wrapper = shallow(<ListItem {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
