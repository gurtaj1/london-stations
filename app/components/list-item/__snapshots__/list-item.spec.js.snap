// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> should render as expected, without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ListItem
    inlineStyleColor="#000000"
    listItem="Circle"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Circle",
      "className": "list-item",
      "style": Object {
        "color": "#000000",
      },
    },
    "ref": null,
    "rendered": "Circle",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Circle",
        "className": "list-item",
        "style": Object {
          "color": "#000000",
        },
      },
      "ref": null,
      "rendered": "Circle",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
